# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/css-typed-om/the-stylepropertymap/declared/get.html.ini
# Commit: 425cbe3c033b
# Full Hash: 425cbe3c033bfc25795979513ebd2bb9fda7338d
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2024-11-28 21:54:11
# Regressor Bug: 1933829
# File Overlap Count: 1
# Description:
#   Bug 1933829 - [wpt-sync] Update web-platform-tests to 2d5d222739fdf91743f8cbc58094976a613194e7, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 2d5d222739fdf91743f8cbc58094976a613194e7
# ==============================================================================

diff -r 732bcd7ac971 -r 425cbe3c033b testing/web-platform/meta/css/css-typed-om/the-stylepropertymap/declared/get.html.ini
--- a/testing/web-platform/meta/css/css-typed-om/the-stylepropertymap/declared/get.html.ini	Wed Nov 27 22:05:28 2024 +0000
+++ b/testing/web-platform/meta/css/css-typed-om/the-stylepropertymap/declared/get.html.ini	Wed Nov 27 22:05:56 2024 +0000
@@ -1,10 +1,4 @@
 [get.html]
-  [Getting a custom property not in the CSS rule returns null]
-    expected: FAIL
-
-  [Getting a valid property not in the CSS rule returns null]
-    expected: FAIL
-
   [Getting a valid property from CSS rule returns the correct entry]
     expected: FAIL
 
@@ -20,3 +14,8 @@
   [Declared StylePropertyMap.get reflects changes in the CSS rule]
     expected: FAIL
 
+  [Getting a custom property not in the CSS rule returns undefined]
+    expected: FAIL
+
+  [Getting a valid property not in the CSS rule returns undefined]
+    expected: FAIL